package.json

```diff
@@ -10,13 +10,13 @@
     "start": "node --import tsx src/index.ts",
     "migrate": "npx prisma migrate deploy",
     "workers": "node --import tsx src/startWorkers.ts",
-    "dev-workers": "cross-env DOTENV_CONFIG_PATH=./.env.development node --import tsx --watch src/startWorkers.ts",
-    "dev-board": "cross-env DOTENV_CONFIG_PATH=./.env.development node --import tsx --watch src/index.ts",
+    "dev-workers": "node --import tsx --watch src/startWorkers.ts",
+    "dev-board": "node --import tsx --watch src/index.ts",
     "dev": "concurrently \"npm run dev-board\" \"npm run dev-workers\"",
-    "import": "node --import=tsx --env-file=.env.development scripts/import-spreadsheet-companies.ts",
-    "test": "DOTENV_CONFIG_PATH=./.env.test jest",
-    "prisma": "dotenv -e .env.development -- prisma",
```

Comment:
We should remove the `dotenv-cli` dependency too since we no longer use the `dotenv` command.